src/parser/open-api/v3/open-api-v3-validator.js

```diff
@@ -6,16 +6,34 @@ const VALIDATOR_HOST = 'https://openapi-converter.herokuapp.com'
  * @param {string} definitionUrl
  * @return {Promise<boolean>}
  */
-export async function validateDefinition (definitionUrl) {
+export async function validateDefinition (definition) {
+  const headers = new Headers({
+    'Accept': 'application/json'
+  })
+
+  const form = new FormData()
+
+  form.append('source', definition)
+
   const url = `${VALIDATOR_HOST}/api/v1/validate`
-  const result = await fetch(`${url}?url=${definitionUrl}`)
+  const fetchOptions = {
+    method: 'POST',
+    headers: headers,
+    body: form
+  }
+  const result = await fetch(`${url}`, fetchOptions)
+
+  if (result.ok === false) {
```

Comment:
Check false booleans with `!`

Reply:
Can this be added to the linting rules? Some people have the habit of using strict checks to avoid implicit conversion issues.

Reply:
You know what, it's fine, keep this line as is.